src/Projection/PdoEventStoreProjector.php

```diff
@@ -930,6 +949,17 @@ private function createLockUntilString(DateTimeImmutable $from): string
         return $from->modify('+' . $secs .' seconds')->format('Y-m-d\TH:i:s') . '.' . $resultMicros;
     }
 
+    private function shouldUpdateLock(DateTimeImmutable $now): bool
```

Comment:
We need this on read model Projection as well

Reply:
oh of course, good catch! Totally missed that.

Reply:
done